common/lib/api-client/middleware/get-cache.js

```diff
@@ -13,12 +13,14 @@ function getCacheMiddleware({ useRedisCache = false } = {}) {
       }
 
       debug('SEARCHING', cacheKey)
-      const data = await cache.get(cacheKey, useRedisCache)
+      const body = await cache.get(cacheKey, useRedisCache)
 
-      if (data) {
-        debug('RETURNING', cacheKey, data)
+      if (body) {
```

Comment:
Is this also different between got and axios?

Reply:
The difference between the two sits on line `20` when we return the response structure.

Got does expect a property called `body` whereas axios would expect `data`. So my thought was to return both properties on the object that is then used in subsequent middleware. This prevents us from having to (and remember to) clear the cache when we toggle the environment variable.

Once we're happy its working okay we can remove support for both and clear the cache to ensure things are all reset but I thought this would be the safest for now.